packages/deployments/contracts/contracts/core/connext/facets/BridgeFacet.sol

```diff
@@ -674,6 +674,13 @@ contract BridgeFacet is BaseConnextFacet {
     // Get the local asset contract address.
     address local = _getLocalAsset(_key, _args.params.canonicalId, _args.params.canonicalDomain);
 
+    // If it is the canonical domain, decrease custodied value
```

Comment:
To save a bit of gas when `bridgedAmt == 0`, this can also be done after 
```solidity
if (_args.params.bridgedAmt == 0) {
      return (0, local, local);
}
```